js/src/views/ParityBar/parityBar.js

```diff
@@ -647,6 +646,13 @@ class ParityBar extends Component {
         return DEFAULT_POSITION;
     }
   }
+
+  setMovingState (moving, extras = {}) {
+    this.setState({ moving, ...extras });
+
+    // Trigger an open event if it's moving
+    this.dispatchOpenEvent(moving);
```

Comment:
It might be useful to distinguish between open for moving and open in the extension in future (for instance to use different animation).